project/.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,6 +1,15 @@
 <!-- THE PR TEMPLATE IS NOT AN OPTION. DO NOT DELETE IT! -->
 
 <!--
+    Show us you choose the right branch.
+    Different branches are used for different things :
+    - 3.x is for everything backwards compatible, like patches and deprecation notices
```

Comment:
Use string interpolation, not directly raw values.

E.g.: This is not true for cache-bundle, or exporter...